### DISCUSSION TITLE
当一个流程图很大时，如何保障调fitView方法后能显示出所有节点？

### DISCUSSION THREAD

**Discussion by DymoneLewis** [didi/LogicFlow#1793] [Q&A] [answered]
如题，这里举个例子
现在画布上有800个节点，分两列展示，调用fitView方法后，整个图只是居中了，没有缩放，如何才能 既缩放又居中？

**Comment by EdwinOlders11** [comment, marked as answer]
正好遇到类似的问题，我发现在节点不算多时调用 _fitView()_，画布又缩放又居中了，但在节点非常多的时候调用就只有居中。
经过一番尝试后发现，画布的默认最小缩放是20%，如果节点很多，缩到最小还是不能放下所有元素， _fitView()_ 就只会居中了。
使用 _lf.setZoomMiniSize(0.01)_ 将最小缩放改为1%或更小后，即使是很大的图形也可以用 _fitView()_ 又缩放又居中了
希望能帮到你
